tests/test_real_world_examples.py

```diff
@@ -36,3 +37,10 @@ def test_webgoat_6_1(self, *_: Any, **__: Any) -> None:
     def test_regression_issue_630(self, *_: Any, **__: Any) -> None:
         with open(join(OWN_DATA_DIRECTORY, 'xml', '1.6', 'regression_issue630.xml')) as input_xml:
             Bom.from_xml(input_xml)
+
+    def test_merged_bom_duplicate_component(self, *_: Any, **__: Any) -> None:
+        with open(join(OWN_DATA_DIRECTORY, 'json', '1.5', 'duplicate_components.json')) as input_json:
+            json = json_loads(input_json.read())
+
+        bom = Bom.from_json(json)
+        self.assertEqual(4, len(bom.components))
```

Comment:
this is a disputable assertion.
the two of the components of the original BOM were duplicated in the first place, why keep them?

Reply:
What is the correct place for this to be disputed, is it part of the implementation or the spec discussion?

Reply:
it is something that could be discussed here: https://github.com/CycloneDX/specification/discussions

Reply:
Where in the spec is it explained what the uniqueness criteria for components are?